### PR TITLE
test additional cohortdata definition tests

### PR DESCRIPTION
**Pull request by UchicagoZchen138**
Jira Ticket: [VADC-19](https://ctds-planx.atlassian.net/browse/VADC-19)

### Improvements

- Additional tests

### CODE REVIEW

File: tests/models_tests/models_test.go [hunk 1]
```diff
@@ -505,3 +505,24 @@ func TestGetCohortDefinitionById(t *testing.T) {
 		t.Errorf("Expected data not found")
 	}
 }
+
+func TestRetrieveDataByOriginalCohortAndNewCohort(t *testing.T) {
+	setUp(t)
+	originalCohortSize := secondLargestCohort.Id
```
Comment (pieterlukasse) [inline tests/models_tests/models_test.go:511]: is this a copy/paste error? Assigning "Id" to a size variable looks strange here

File: tests/controllers_tests/controllers_test.go [hunk 2]
```diff
@@ -747,6 +747,44 @@ func TestRetrievePeopleIdAndCohort(t *testing.T) {
...
+		int64(1): {
+			"ID_4_5": "NA",
+		},
+		int64(2): {
+			"ID_4_5": "NA",
+		},
+		int64(3): {
+			"ID_4_5": "NA",
+		},
+	}
```
Comment (pieterlukasse) [inline tests/controllers_tests/controllers_test.go:778]: this is the scenario where persons are not part of any of the cohorts right? What about the scenario where they are part of both cohorts? I.e. just create a cohort pair like
```go
cohortPairs := [][]int{
		{3,3},
	}
```
Other than that is a strange/non-realistic scenario, I think this should return NA as well.

**Review by pieterlukasse — COMMENTED** [review]
thanks @UchicagoZchen138 ! Please see my two comments.

File: tests/models_tests/models_test.go (hunk 1 above)
Comment (UchicagoZchen138) [reply tests/models_tests/models_test.go]: yup good catch, should have been `secondLargestCohort.cohortSize`

**Review by pieterlukasse — APPROVED** [review]

**Pull request merged**